flow/designs/nangate45/aes/config.mk

```diff
@@ -7,6 +7,7 @@ export SDC_FILE      = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
 
 # These values must be multiples of placement site
 # x=0.19 y=1.4
-export DIE_AREA    = 0 0 620.15 620.6
-export CORE_AREA   = 10.07 11.2 610.27 610.8
+export DIE_AREA    = 0 0 249.85 250.6 
+export CORE_AREA   = 10.07 11.2 239.97 240.8 
 
+export PLACE_DENSITY_LB_ADDON = 0.20
```

Comment:
Can we just change the default or are there designs that can't be run with this increase?

Reply:
There is a default PLACE_DENSITY setting in the config  which is a static number -- The LB_ADDON settings over rides the PLACE_DENSITY value -- It is better is it makes the PLACE_DENSITY a function of the floorplan utilization  It is probably better to make the LB_ADDON (stands for lower bound addon) the default in the config file and have users reset it with the absolute PLACE_DENSITY setting as necessary

Reply:
LB_ADDON was created for the autotuner so that it could be more pdk agnostic.   If we want to make it the default then we should give it a more meaningful name and switch over rather than having a mishmash.